build/media-suite_compile.sh

```diff
@@ -1650,11 +1655,13 @@ if [[ $vvc = y ]] &&
     do_uninstall bin-video/vvc
     # patch for easier install of apps
     # probably not of upstream's interest because of how experimental the codec is
-    do_patch "https://0x0.st/sG0V.txt"
+    do_patch "https://0x0.st/sG0V.txt" am
+    do_patch "https://gist.githubusercontent.com/1480c1/48a0440bf6fd42ba4e41d833b71b9682/raw/0001-CMake-add-USE_CCACHE-variable-to-disable-using-found.patch" am
```

Comment:
@wiiaboo In cases like this, do you want me to upload the patch to 0x0.st? Since I'm not sure if I will ever need to rebase the patch and also it probably will be neigh impossible to upstream the patch (it seems that you are required to be a JVET team member or related to even fork on their server and open a merge request).

Reply:
No opinion. 0x0.st is fine, unless it breaks frequently in which case gist instead.
(Does anyone even use VVC yet? Aren't MPEG codecs dead? Why would anyone want a patent-encumbered codec after AV1 and derivatives proliferate enough?)